core/src/main/java/org/elasticsearch/plugins/PluginCli.java

```diff
@@ -40,7 +39,20 @@ public PluginCli(Environment env) {
     }
 
     public static void main(String[] args) throws Exception {
-        BasicConfigurator.configure(new NullAppender());
+        // initialize default for es.logger.level because we will not read the logging.yml
```

Comment:
We could also fire the logs through a log4j appender that targets our Terminal object. I don't know that it'd buy us too much. Maybe in a way that I haven't wrapped my head around yet.